src/ua.c

```diff
@@ -2029,6 +2029,29 @@ int ua_print_allowed(struct re_printf *pf, const struct ua *ua)
 }
 
 
+/**
+ * Print the supported events header
+ *
+ * @param pf  Print function
+ * @param ua  User-Agent object
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int ua_print_allowed_events(struct re_printf *pf, const struct ua *ua)
```

Comment:
perhaps move this function to account.c ?

Reply:
Initially I had that function in another file, but is felt so lonely there between all those dissimilar looking functions that I moved it to its current place between its ua_print_* friends. But I happily move it if deemed necessary.